tasmota/tasmota_xsns_sensor/xsns_33_qmc5883l.ino

```diff
@@ -24,19 +24,20 @@
  *
  * I2C Address: 0x0D
  * 
- * #define QMC5883L_OVERSAMPLE           1    // 0 .. 3 => 512, 256, 128, 64
- * #define QMC5883L_GAUSS                1    // 0,1 => 2GAUSS, 8GAUSS
- * #define QMC5883L_FILTER               2    // 0 .. 3 => 10HZ, 50HZ, 109HZ, 200HZ
- * #define QMC5883L_TEMP_SHIFT           23   // sensor temperature are not calibrated (only relativ measurement) and need an absolute ground value in °C (see datasheet)
- *
- * Calibration :
- * valid relations :      1 T = 10000 G ... 1G = 0.1mT = 100µT
- * Dynamic Output Field Range ±2 or ±8 Gauss ==> ±200µT or ±800µT
- * Sensitivity [1]
- *   Field Range = ±2G 12000 LSB/G  --> factor = 120
- *   Field Range = ±8G 3000 LSB/G  --> factor = 30
+ * #define QMC5883L_OVERSAMPLE  0     // 0 .. 3 => over Sample Ratio : 512, 256, 128, 64
+ * #define QMC5883L_GAUSS       1     // 0 .. 1 => full Scale : 2GAUSS, 8GAUSS
+ * #define QMC5883L_FILTER      0     // 0 .. 3 => Output Data Rate : 10HZ, 50HZ, 109HZ, 200HZ
```

Comment:
Functional change?

Reply:
no ... unchanged

Reply:
0 => 2 ?

Reply:
an additional compile option : #define MC5883L_DECIMAL to modify displaying dcimals

Reply:
As eku said: 2 => 0 ? and 1 => 0 ?

Reply:
the driver reads the registers once a second - so providing them 10 times a second is more than sufficient (2=>0). That reduces the power consumption of the chip and allows longer filtering (1=<0)